ui/lib/pki/addon/components/page/pki-configuration-edit.hbs

```diff
@@ -3,6 +3,25 @@
     <AlertBanner @type="danger" @message={{this.errorBanner}} data-test-error-banner />
   {{/if}}
   <form {{on "submit" (perform this.save)}}>
+    <fieldset class="is-shadowless is-marginless is-borderless is-fullwidth" data-test-cluster-config-edit-section>
```

Comment:
nit: can we remove the margin-top on the fieldset? not a necessary change :) 
<img width="1090" alt="Screenshot 2023-05-23 at 11 50 58 AM" src="https://github.com/hashicorp/vault/assets/30884335/e16e93b3-2061-478e-acb7-1e91975f8e09">